test/cufinufft3d1_test.cu

```diff
@@ -185,7 +185,14 @@ int main(int argc, char* argv[])
 
 	cudaFreeHost(x);
 	cudaFreeHost(y);
+	cudaFreeHost(z);
```

Comment:
ah, yes, the joys of C-style mem mgmt!  But this is only a test code, not the main lib.